src/main/java/com/duckblade/osrs/dpscalc/calc/gearbonus/VampyreGearBonus.java

```diff
@@ -1,33 +1,60 @@
 package com.duckblade.osrs.dpscalc.calc.gearbonus;
 
+import com.duckblade.osrs.dpscalc.calc.WeaponComputable;
 import com.duckblade.osrs.dpscalc.calc.compute.ComputeContext;
 import com.duckblade.osrs.dpscalc.calc.compute.ComputeInputs;
+import com.duckblade.osrs.dpscalc.calc.model.DefenderAttributes;
 import com.duckblade.osrs.dpscalc.calc.model.GearBonuses;
+import com.google.common.collect.ImmutableMap;
+import lombok.RequiredArgsConstructor;
+import net.runelite.api.ItemID;
+
 import javax.inject.Inject;
 import javax.inject.Singleton;
-import lombok.RequiredArgsConstructor;
+import java.util.Map;
+
+import static com.duckblade.osrs.dpscalc.calc.compute.ComputeInputs.ATTACK_STYLE;
 
-// TODO
 @Singleton
 @RequiredArgsConstructor(onConstructor = @__(@Inject))
 public class VampyreGearBonus implements GearBonusComputable
```

Comment:
Since this file only deals with vampyrebane weapons in this state, and doesn't handle any immunities or reductions, it should be renamed to `VampyreBaneGearBonus`